.github/workflows/trigger_frontend_template_releases.yml

```
@@ -0,0 +1,38 @@
+          - generiekzaakafhandelcomponent/gzac-frontend-template
+          - generiekzaakafhandelcomponent/gzac-evenementenvergunning-demo-frontend
+          - valtimo-platform/valtimo-frontend-template
+    steps:
+      - name: Determine target branch
+        id: branch
+        run: |
+          MAJOR=$(echo "${{ inputs.version }}" | cut -d. -f1)
+          echo "ref=v/${MAJOR}" >> "$GITHUB_OUTPUT"
+
+      - name: Trigger release workflow
+        if: ${{ secrets.TEMPLATE_RELEASE_PAT != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)